core/modules/replica/ChunksApp.cc

```diff
@@ -41,14 +41,16 @@ using namespace std;
 
 namespace {
 
+using namespace lsst::qserv::replica;

```

Comment:
I'm not sure if this is the right thing to do as per the de-jure LSST Style Guide or Qserv de-factor one. Note that this file has the following structure:
```
namespace {
    using namespace lsst::qserv::replica;
    void foo() { ... }
}
namespace last {
namespace qserv {
namespace replica {
void ChunkApp::bar() {
  ::foo();
}}}
```

Reply:
Declarations inside anonymous namespace are visible in a global namespace too. Placing `using` inside anonymous namespace may give someone the idea that it only applies to that namespace which is not true. Move it outside to avoid this confusion.

Reply:
I didn't know that. Thanks! Every time I'm asking you to review my PR I'm learning something new.